pages/tutorials/3.0/system/time.md

```diff
@@ -7,13 +7,18 @@
 
 ## Time in SFML
 
-Unlike many other libraries where time is a uint32 number of milliseconds, or a float number of seconds, SFML doesn't impose any specific unit or type for time values. Instead it leaves this choice to the user through a flexible class: [`sf::Time`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Time.php "sf::Time documentation"). All SFML classes and functions that manipulate time values use this class.
+Unlike many other libraries where time is a uint32 number of milliseconds, or a float number of seconds, SFML doesn't impose any specific unit or type for time values.
+Instead it leaves this choice to the user through a flexible class: [`sf::Time`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Time.php "sf::Time documentation").
```

Comment:
> I stead it leaves this choice to the user

This is not true. We dictate the use of `int64` number of microseconds as the underlying representation. `std::chrono::duration` by comparison _does_ let users pick. To let users pick you have to use templates. The language isn't harmful per se, but it's not correct. 

More broadly speaking, while upgrading the docs to v3 do we also want to go through them this finely to fix wording? Or, would we rather just do the minimal requires to make the docs in line with the v3 API and go fix wording later? I'm open to both approaches but am leaning towards the latter. Let's for now focus on making the docs up to date with v3. At a later time (potentially after v3 ships) we can continue to refine this wording to make the docs as good as possible.

Reply:
What the sentence tries to communicate however isn’t wrong. It’s not talking about the underlying implementation, but it talks from a user perspective. The user doesn’t need to use `int64` in their code, but they can use `sf::Time` instead, that’s what the phrase tried to say.

—-

I have a feeling that if we try to additionally phrase everything “better”, we could end up in some rabbit holes and discussions, that would slow down the process more and delay the initial goal. Of course if there’s some glaring mistake, do correct it.